WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -740,9 +742,20 @@ class MySiteViewModel @Inject constructor(
         postCardsSource.refresh()
     }
 
+    fun onPostItemClick(postId: Int) {
```

Comment:
Minor (🔍): Consider making this function `private`.

PS: The same applies to the below `onPostCardFooterLinkClick` function.

Reply:
In d33221f